lib/jobba/status.rb

```diff
@@ -6,6 +6,14 @@ class Status
 
     include Jobba::Common
 
+    def self.all
+      Query.new
+    end
+
+    def self.where(*args)
+      all.where(*args)
+    end
```

Comment:
The one reason to have `all` and `where` on the Status class would be to have an API similar to ActiveRecord.

`find` and `where` are related in my head (`find(id)` being just `where(id: id).first`), so it's odd to me to have them in separate classes.